### PR TITLE
fix: Remove divider to resolve UI jumping in session overview card

### LINKED ISSUE #5343: Public Schedule: Color area "jump difference" when extending area
On the public schedules, if the user expands a session the color area "jumps" a bit. The size of the color area before and after the expansion is different. Expected: Area should always be the same.

Compare https://eventyay.com/e/8fa7fd14/schedule

Screenshot from 2020-10-21 16-11-42

### PR DESCRIPTION
**Pull request by Abhijay007**
Color area "jump difference"  issue resolved #5343

Fixes #5343

#### Short description of what this resolves:
In the public schedules, if the user expands a session the color area "jumps" a bit. The size of the color area before and after the expansion is different. This PR resolves it. Now Area should always be the same.

#### Changes proposed in this pull request:

- Removed Unwanted CSS (Ui divider)
- Added required margin and resolved the jump issue.

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

after

### CODE REVIEW

File: app/components/public/session-item.hbs [hunk 1]
```diff
@@ -83,8 +83,7 @@
     {{/if}}
     
     {{#each @session.speakers as |speaker|}}
-      <div class="ui divider"></div>
```
Comment (maze-runnar) [inline app/components/public/session-item.hbs:86]: what if there are more than 1 speaker? Without a divider. How it looks? share a screenshot.

Comment (Abhijay007) [reply app/components/public/session-item.hbs:86]: withoutDiv

it looks like this

**Comment by mariobehling** [comment]
Testing on the autodeployed test link shows the issue still exists: https://open-event-frontend-git-development.eventyay.vercel.app/e/8fa7fd14/schedule

**Comment by iamareebjamal** [comment]
Don't know why vercel commented that link but the preview is here - https://open-event-frontend-g1ui3gar2.vercel.app/e/8fa7fd14/schedule

**Comment by Abhijay007** [comment]
> Testing on the autodeployed test link shows the issue still exists: https://open-event-frontend-git-development.eventyay.vercel.app/e/8fa7fd14/schedule

sir, please check the below link it works. commented by @iamareebjamal

**Comment by Abhijay007** [comment]
> Don't know why vercel commented that link but the preview is here - https://open-event-frontend-g1ui3gar2.vercel.app/e/8fa7fd14/schedule

thanks for sharing the valid link

**Review by mariobehling — APPROVED** [review]
Nice, thank you!

**Review by Raunaksingh100 — APPROVED** [review]

**Comment by iamareebjamal** [comment]
@Abhijay007 Please see the new PR title and try to make future PR titles like this

**Comment by Abhijay007** [comment]
> @Abhijay007 Please see the new PR title and try to make future PR titles like this

Ok sir, And thanks for your kind suggestion 😊.

**Pull request merged**